### PR TITLE
Monkey HUD funtimes, lightbulb nullspacing

### LINKED ISSUE #12774: Flaming skull (plasmaman monkey) issues
#### Description of issue

They have no hands but still have swap, drop, throw HUDs
Light bulbs grabed by them get null'd
They can't attack other simple mobs (tested with spiderling and mouse)

https://webmshare.com/play/Dm40G

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by ghost**
Reworks monkey HUD to have more variability (Swap button doesn't exist if you've only got 1 hand or less, throw and drop buttons don't exist if you have no hands)

More fun variables to modify your monkey with (Boolean to control if it can wear a mask, or a backpack)

Changes the random emotes monkeys make when they have no client to a proc, so it can be controlled better (Basically removing the tail wagging from those that lack tails physically)

Turns out light removal with your bare hands never checked if the light was put into your hands, and would just keep the light in nullspace. Now, it just drops the light on the floor next to the subject if they have no hands with which to hold it.

Fixes #12774

:cl:

* bugfix: skeleton and plasmamen monkeys no longer nullspace light tubes
* bugfix: Makes monkey HUD have more dependencies, so no swap button if the monkey only has one or less hands, etc.
* bugfix: Monkeys without tails should now no longer wag their invisible tails

### CODE REVIEW

**Comment by Probe1** [comment]
Put the Gray stuff in another PR.  Or this is gonna sit longer than a bug fix should.

File: code/modules/mob/living/carbon/monkey/skeleton.dm [hunk 1]
```diff
@@ -8,9 +8,12 @@
 	canWearClothes = 0
 	canWearHats = 0
 	canWearGlasses = 0
+	canWearBack = 0
+	canWearMasks = 0
 	languagetoadd = LANGUAGE_CLATTER
 	brute_damage_modifier = 2
 	movement_speed_modifier = 0.23
+	safe_oxygen_min = 0
```
Comment (JustSumBody) [inline code/modules/mob/living/carbon/monkey/skeleton.dm:16]: This isn't mentioned in the PR description

File: code/modules/mob/living/carbon/monkey/skeleton.dm [hunk 2]
```diff
@@ -35,4 +41,5 @@
 	light_range = 2
 	light_power = 0.5
 	light_color = "#FAA019"
+	burn_damage_modifier = 0.33
```
Comment (JustSumBody) [inline code/modules/mob/living/carbon/monkey/skeleton.dm]: Nor is this

**Review by JustSumBody — COMMENTED** [review]
something something atomic

File: code/modules/mob/living/carbon/monkey/skeleton.dm (hunk 1 above)
Comment (ghost) [reply code/modules/mob/living/carbon/monkey/skeleton.dm:16]: To be fair, skeletons not needing to breath, but their disembodied heads needing to just didn't sit right

**Pull request merged**